public/js/livesearch.js

```diff
@@ -0,0 +1,104 @@
+	//init vars
+	var input;
+	var results;
+	var indiResults;	
+	var urlResult;
+	var formMethod;
+	var typeData;
+	var resultFilter;
+	var identifier ;
+
+	//set values to vars, this is done wherever the livesearch is called
+	function setInputListener(inputField , resultField, resultHolder, uUrl, method, typeD){
+		
+		input = inputField;
+		results = resultField;
+		indiResults = resultHolder;
+		urlResult =  uUrl + '/';
+		typeData = typeD;
+		return this;
+	}	
+	
+	//set the method type for ajax submission
+	function setMethod(method){
+		var regex= /(GET|PUT)/g;
+		var methods = method.match(regex);
+		if(methods != null){
+			return this;
+		}
+		console.log("The method was not set correctly");
+		return false;
+	}
+
+	//setting the filter / find for returning only desired result html
+	function setResultFilter(filter){
+		resultFilter = filter;
+		return this;
+	}
+	
+	//debugging
+	//console.log("live search is loaded");
```

Comment:
Remove this commented out code (unless you think you'll need it here later). Any place you have commented out debugging code I think should be removed to clean up the file.